packages/components/charts/src/radar-chart.tsx

```diff
@@ -98,6 +98,8 @@ export const RadarChart = forwardRef<RadarChartProps, "div">((props, ref) => {
     tooltipAnimationDuration,
     unit,
     valueFormatter,
+    polarAngleAxisTickFormatter = valueFormatter,
+    polarRadiusAxisTickFormatter = valueFormatter,
```

Comment:
Is it necessary to assign a `valueFormatter`?

Reply:
Fixed.